config/dependencies/authorino/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
+- github.com/Kuadrant/authorino-operator/config/default?ref=main
```

Comment:
Note, these dependency kustomizations are referencing the remote kustomization for that component rather than cloning and copying or expecting a generated set of manifests to exist in that repo.
Having the kustomizations work this way allows modifications to remote sets of manifests that aren't directly the responsibility of this repo. For example it would be possible for us to override the image being used for the authorino manager deployment in the same we can modify this operator manage image.

```
$ cd config/dependencies/authorino && ../../../bin/kustomize edit set image quay.io/3scale/authorino-operator=quay.io/mnairn/authorino-operator:myfeature
$ cd ../../..
$ ./bin/kustomize build config/dependencies/authorino/ | grep image: 
              image:
        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
        image: quay.io/mnairn/authorino-operator:myfeature
```

Reply:
@guicassolato If we were to take this approach with authorino and the authorino-operator would this get round potential issues you had with pinning the version of authorino on the authorino-operator main branch? i.e. If you want to install the latest version of authorino using the latest version of the operator, add a make target to authorino that overrides the authorino image and sets it to `quay.io/kuadrant/authorino:latest`